dags/databricks-ml-retrain-example.py

```diff
@@ -34,28 +40,43 @@ def databricks_ml_retrain_example():
         },
         do_xcom_push=True
     )
-
     
     @task()
     def register_model(databricks_run_id: str):
-    
+        """Register model in MLflow
+
+        Uses the run_id to get the notebook output which contains the model URI needed to register the model.
+
+        Returns the new model version
+
+        Keyword arguments:
+        databricks_run_id -- run_id of the training notebook used in the "train" task.
+        """
+
         logging.info(f'Training notebook run_id: {databricks_run_id}')
 
+        # Get model URI from notebook output
         databricks_hook = DatabricksHook()
         model_uri = databricks_hook.get_run_output(databricks_run_id)['notebook_output']['result']
         logging.info(f'Model URI: {model_uri}')
-        
-        model_version = mlflow.register_model(model_uri, model_name)
 
-        logging.info(f'Name: {model_version.name}')
-        logging.info(f'Version: {model_version.version}')
+        # Register the model get new version number
+        registered_model = mlflow.register_model(model_uri, model_name)
 
-        return model_version.version
-    
+        return registered_model.version
 
     @task()
-    def submit_for_approval_to_stage(model_version, databricks_instance):
+    def submit_for_approval_to_stage(model_version, databricks_instance: str):
+        """Submit transition to Staging request
+
+        After registration a transition to Staging request is submitted which can be reviewed by users.
+
+        Returns response from MLflow API for the request.
 
+        Keyword arguments:
+        model_version -- Version of the model that should be submitted with transition request.
```

Comment:
nit: inconsistent use of periods  in docstrings (this is in a few other places, too)